### PR TITLE
Add detailed documentation

### PR DESCRIPTION
**Pull request by ParadoxOrigins**
**This pull request shouldn't be merged directly**

There may be things to correct, as much as some of the documented methods outputs aren't implemented in _MalScraper_ yet

[EDIT] Indeed, forgot that it is possible to use `getEpisodesList()` without the ID (i will get back to it later)

### CODE REVIEW

**Comment by xdk78** [comment]
This project using standard code style, so remove the semicolons from code in Readme :grinning:

**Comment by ParadoxOrigins** [comment]
heck, ESLint, 2018, please

**Comment by Kylart** [comment]
That is an awesome work!

> This project using standard code style, so remove the semicolons from code in Readme 😀

He's actually right, semicolons should be removed 😄

> [EDIT] Indeed, forgot that it is possible to use getEpisodesList() without the ID (i will get back to it later)

I think that's the only thing left since I implemented the changes we talked about.

**Comment by ParadoxOrigins** [comment]
_angry murlock noises_

Alright the semicolons will be removed then, i'll pack the changes tonight/tomorrow

**Comment by ParadoxOrigins** [comment]
oh my bad the commit name reference the first issue, still have troubles with github after one year ;-;

File: README.md [hunk 1]
```diff
@@ -46,126 +72,543 @@ Any contribution is welcomed.
...
-// Second way, you have only the anime's name
-// Longer that forst way since it makes two requests
-// to get the best results according to the name.
-malScraper.getEpisodesList('Sakura Trick')
-  // `data` is an array containing the episodes and
-  // their information.
+//Alternatively, if you only have the name and not the id, 
+//you can let the method fetch the id on the way at the cost of being slower
+
+const name = "Sakura Trick";
```
Comment (Kylart) [inline README.md]: double-quotes are prohibited unless there is a real need for it.

**Pull request merged**